dashboard/modules/metrics/dashboards/serve_deployment_grafana_dashboard_base.json

```diff
@@ -62,7 +97,7 @@
         "name": "Deployment",
         "options": [],
         "query": {
-          "query": "label_values(ray_serve_deployment_replica_healthy{{{global_filters}}}, deployment)",
+          "query": "label_values(ray_serve_deployment_replica_healthy{{application=~\"$Application\",{global_filters}}}, deployment)",
```

Comment:
do we need to scope the "Replica" variable to the selected Application as well?

Reply:
Yeah good catch, will update!